### DISCUSSION TITLE
Allow all plots to be displayed with or without the background value

### DISCUSSION THREAD

**Discussion by smk78** [SasView/sasview#2203] [Ideas]
Pursuant to closing https://github.com/SasView/sasview/issues/1614 as won't fix, @RichardHeenan suggested that it would be beneficial to provide a mechanism (eg, radio button) by which all relevant plots in SasView could be displayed with or without the background value included.

**Comment by lucas-wilkins** [comment]
There are quite a lot of questions about how this should look and be implemented. Also, what about plots of models.
